### DISCUSSION TITLE
Notas entrega de proyecto

### DISCUSSION THREAD

**Discussion by Fonsydo** [rramosp/ai4eng.v1#256] [00 - General]
Saludos.

Tengo una duda con respecto a las notas de los proyectos, ya que estas no se encuentran reflejadas en la plataforma del curso. En la plataforma sólo salen notas de los laboratorios y del quiz.

Muchas gracias.

**Comment by Felipe-RA** [comment]
Saludos @Fonsydo 

Por favor indicame tu correo electrónico para echarle un vistazo

**Reply by Fonsydo** [reply]
fonsy.mercado@udea.edu.co

**Reply by Felipe-RA** [reply]
El motivo por el que no tienes la nota es porque no rellenaste el formulario listando tu repositorio de github según las pautas estipuladas en la página del curso.

Recuerda que con una penalización del 50% podemos revisartelo.

¡Hasta pronto!

**Reply by Fonsydo** [reply]
Es muy raro el asunto, porque recuerdo haber seguido todas las instrucciones al pie de la letra y recuerdo haber llenado el formulario de Google en el que se registraba el proyecto.

Igual si no hay nada más que hacer si prefiero que se califique con la penalización a que no lo hagan.

**Reply by Fonsydo** [reply]
¿Para que me lo revisen con la penalización debo llenar de nuevo el formulario?

**Reply by imEag** [reply]
No es necesario, nosotros nos encargamos de revisarlo

**Reply by Fonsydo** [reply]
Nuevamente estoy sin nota, ahora en la Entrega 2

**Reply by imEag** [reply]
ya está arreglado
